homotopy-web/src/model/proof/signature.rs

```diff
@@ -148,7 +150,7 @@ impl Signature {
     ) -> Result<Diagram, NewDiagramError> {
         let id = self.next_generator_id();
         let generator = Generator::new(id, source.dimension() + 1);
-        let diagram = DiagramN::from_generator(generator, source, target)?;
+        let diagram = DiagramN::from_generator(generator, source, target, false)?;
```

Comment:
@NickHu If you merge the PR into your branch, set this to `true` to make every generator framed until we figure out a way in the UI to toggle that. In the meantime, I will leave this as `false` so we can test things like Reidemeister 1 in the preview.